### PR TITLE
Issue #3

### PR DESCRIPTION
**Pull request by elefher**

### CODE REVIEW

File: __tests__/http/controllers/AccountTypesController.test.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+AccountTypeService.update = jest.fn()
+AccountTypeService.fetchAll = jest.fn()
+AccountTypeService.fetchById = jest.fn()
+AccountTypeService.deleteById = jest.fn()
+
+const errorMessage = 'Error message'
+const BoomNotFoundError = Boom.notFound(errorMessage)
+const BoomBadDataError = Boom.badData(errorMessage)
+
+let req, res, next
```
Comment (GeoDoo) [inline __tests__/http/controllers/AccountTypesController.test.js:17]: It is better to wrap things inside a `describe` block so you do not pollute the global namespace. So, I would wrap everything inside a describe like these vars and the beforeEach

File: __tests__/http/controllers/AccountTypesController.test.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  next = jest.fn().mockImplementation(err => {
+    return Boom.isBoom(err)
+      ? res.status(err.output.statusCode).json(err.output.payload)
+      : res.status(err.status || 500).json({ message: err.message })
+  })
+})
+
+describe('Create account types', () => {
+  it('should have a create function', () => {
+    expect(typeof AccountTypes.create).toBe('function')
```
Comment (GeoDoo) [inline __tests__/http/controllers/AccountTypesController.test.js:32]: do you care testing this in general? I wouldn't, also it gets tedious too. 

Try not to test all implementation details

**Review by GeoDoo — APPROVED** [review]
All in all looks good to me, but think about not testing submodules. For example, if you remove Boom in the future (let's say you want to refactor the code) I suppose your tests will fail, because you test Boom heavily on your unit tests, right?

File: __tests__/http/controllers/AccountTypesController.test.js (hunk 1 above)
Comment (elefher) [reply __tests__/http/controllers/AccountTypesController.test.js:17]: ok

File: __tests__/http/controllers/AccountTypesController.test.js (hunk 2 above)
Comment (elefher) [reply __tests__/http/controllers/AccountTypesController.test.js:32]: yep pointless

**Comment by elefher** [comment]
> All in all looks good to me, but think about not testing submodules. For example, if you remove Boom in the future (let's say you want to refactor the code) I suppose your tests will fail, because you test Boom heavily on your unit tests, right?

correct I should change it to make it more generic and not check that the response is Boom.

**Review by GeoDoo — APPROVED** [review]

**Pull request merged**